### PR TITLE
chore: New atPlatform logo

### PR DESCRIPTION
**Pull request by cpswan**
**- What I did**

Replaced @‎platform logo with atPlatform logo

**- Description for the changelog**

chore: New atPlatform logo

### CODE REVIEW

File: at_ve_doctor/README.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
-<img width=250px src="https://atsign.dev/assets/img/@platform_logo_grey.svg?sanitize=true">
+<img width=250px src="https://atsign.dev/assets/img/atPlatform_logo_gray.svg?sanitize=true">
 
 ## at_ve_doctor
 
 at_ve_doctor is a simple utility than can check the state of the secondaries
 running in the virtual environment (VE).  
-The virtual environment provides the full @platform stack including a @root
-server and a number of preconfigured @signs.
+The virtual environment provides the full atPlatform stack including a atRoot
```
Comment (gkc) [inline at_ve_doctor/README.md]: change 'a atRoot' to 'an atRoot'

File: at_cram/README.md [hunk 2]
```diff
@@ -32,11 +32,9 @@ at_cram tool and a digest given back. This digest in then entered using the cram
 as the @colin🛠@ prompt is returned.
 
 ## How this works
-The shared secret is know to the person and the secondary but we do not want to send it across the wire.
-The solution is adding the shared secret to the challenge and then sending over a SHA hash of the resulting 
-string. This is what the at_cram binary does, it is not complex at all, but very effective.
-
 
+The shared secret is know to the person and the secondary but we do not want to send it across the wire.
```
Comment (gkc) [inline at_cram/README.md]: change 'is know' to 'is known'

**Review by gkc — CHANGES_REQUESTED** [review]

File: at_ve_doctor/README.md (hunk 1 above)
Comment (cpswan) [reply at_ve_doctor/README.md]: Fixed by [1064c4f](https://github.com/atsign-foundation/at_tools/pull/202/commits/1064c4f8a953913e7ebb14c99f810a5a43f4154c)

File: at_cram/README.md (hunk 2 above)
Comment (cpswan) [reply at_cram/README.md]: Fixed by [1064c4f](https://github.com/atsign-foundation/at_tools/pull/202/commits/1064c4f8a953913e7ebb14c99f810a5a43f4154c)

**Review by gkc — APPROVED** [review]

**Pull request merged**